src/inet/tests/BUILD.gn

```diff
@@ -61,7 +61,8 @@ chip_test_suite("tests") {
     "TestInetEndPoint",
   ]
 
-  if (current_os != "mac" && chip_device_platform != "esp32") {
+  if (current_os != "mac" && chip_device_platform != "esp32" &&
+      current_cpu == "x64") {
```

Comment:
a comment here may be useful - it is not immediately clear that x64 CPU means 'not rasberry pi'

Reply:
Done.